recipes/sym/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "sym" %}
+{% set version = "0.3.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sym-{{ version }}.tar.gz
+  sha256: 80358157e65ef7d34b93fe8df58db785e5b817c1d0a0e15a2578d2694c09d630
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - numpy
+    - python
+    - pytest
```

Comment:
Is pytest really a runtime dep?

Reply:
I'm organizing a lot of packages, I may have been wrong. I'll check.

Reply:
> Is pytest really a runtime dep?

You're right. Are you responsible for reviewing the commits? Should I edit and redo the commit?

Reply:
Yes & yes. :wink: I am a member of the @conda-forge/staged-recipes team and also you should make and edit and push up a new commit